docs/configuration/ai-analysis.rst

```diff
@@ -299,36 +299,318 @@ Advanced - Customizing HolmesGPT
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 
-Adding Custom Tools to Holmes
+Holmes Toolsets
 -------------------------------------
 
-Holmes allows you to define custom toolsets that enhance its functionality by enabling additional tools to run Kubernetes commands or other tasks.
+Holmes allows you to define tools that enhance its functionality by enabling additional tools to run Kubernetes commands or other tasks.
+
+Default Toolsets in Holmes
+--------------------------
+Holmes comes with a set of default toolsets that provide tools out-of-the-box. Some of these toolsets are enabled by default such as interacting with Kubernetes resources, or fetching logs and metrics.
+Full list can be found `here <https://github.com/robusta-dev/holmesgpt/tree/master/holmes/plugins/toolsets>`_
 
-In this guide, we will show how to add a custom toolset to Holmes in your ``generated_values.yaml`` file.
+By default, any of the toolsets may be disabled. You can enable or disable them by modifying the ``generated_values.yaml`` file.
 
 .. code-block:: yaml
+    enableHolmesGPT: true
+    holmes:
+      additionalEnvVars:
+        - name: ROBUSTA_AI
+          value: "true"
+      toolsets:
+        kubernetes/logs:
+          enabled: false
+
+After making changes, apply them using Helm:
+
+.. code-block:: bash
 
+    helm upgrade robusta robusta/robusta --values=generated_values.yaml --set clusterName=<YOUR_CLUSTER_NAME>
+
+
+Additionally, you can overwrite any of the fields in the default toolsets, such as updating the tools, prerequisites, or descriptions, to customize them for your specific needs.
+For example:
+
+.. code-block:: yaml
     enableHolmesGPT: true
     holmes:
       additionalEnvVars:
         - name: ROBUSTA_AI
           value: "true"
       toolsets:
-        # Name of the toolset (for example "mycompany/internal-tools")
-        # Used for informational purposes only (e.g. to print the name of the toolset if it can't be loaded)
-        - name: "resource_explanation"
-          # List of tools the LLM can use - this is the important part
-          tools:
-          # Name is a unique identifier for the tool
-            - name: "explain_resource"
-              # The LLM looks at this description when deciding what tools are relevant for each task
-              description: "Provides detailed explanation of Kubernetes resources using kubectl explain"
-              # A templated bash command using Jinja2 templates
-              # The LLM can only control parameters that you expose as template variables like {{ resource_name }}
-              command: "kubectl explain {{ resource_name }}"
+      kubernetes/logs:
```

Comment:
Lets show a real example here. @arikalon1 any ideas on something useful to show here?

Reply:
I added more realistic example with enhanced functionality of confluence